src/components/Model3DViewer/stories/basic.story.mdx

```diff
@@ -0,0 +1,147 @@
+import { Meta, Story, Preview, Props } from '@storybook/addon-docs/blocks';
+import { Model3DViewer } from '../Model3DViewer';
+import { 
+  decorators,
+  modelId,
+  revisionId,
+  onClick,
+  onProgress,
+  onComplete,
+  onReady,
+  cache,
+  onScreenshot,
+  slice,
+  slider
+} from './helper';
+
+<Meta title="Others|Model3DViewer" decorators={decorators} />
+
+# Model3DViewer
+
+This component can be used to visualize 3D models from Cognite Data Fusion.
+The component will visualize the full 3D model by default, but by providing an optional `boundingBox` parameter, you can download only a specific area. This could for instance be around a specific asset.
+
+To retrieve a 3D model you need to provide:
+
+- `modelId` – you can get it via SDK call – `Models3DAPI.list()`. You receive an iterator as a result, which could be used to get list of model ids associated to your tenant. 
+- `revisionId` – you can get it via SDK call `Revisions3DAPI.list(modelId)`. You receive an iterator as a result, which could be used to get list of revision ids available for provided modelId
+
+**NOTE:** The component should have `ClientSDKProvider` as a parent component in react component tree.
+
+#### Requirements:
+
+To use this component you have to install `@cognite/3d-viewer` package (version 5.x.x):
+
+```bash
+yarn add @cognite/3d-viewer@5
+```
+
+## Basic Usage:
+
+Imports:
+
+```jsx
+import 'antd/dist/antd.css';
+
+import React from 'react';
+
+import {
+  Cognite3DViewer,
+  Cognite3DModel,
+  OnProgressData,
+} from '@cognite/3d-viewer';
+import { Revision3D } from '@cognite/sdk';
+import { Model3DViewer } from '@cognite/gearbox';
+
+```
+
+<Preview>
+  <Story name="Basic Usage">
+    <Model3DViewer
+      modelId={modelId}
+      revisionId={revisionId}
+      onClick={onClick}
+      onProgress={onProgress}
+      onComplete={onComplete}
+      onReady={onReady}
+      cache={cache}
+    />
+  </Story>
+</Preview>
+
+## Available props
+
+<Props of={Model3DViewer} />
+
+##### `assetId` prop explanation
+
+This prop is needed to control highlighting of asset node in the model right after init process or by changing `assetId` prop after viewer has been initialized.
```

Comment:
please move this inside `<Props />` I don't see a point of having it here separately. Just replace current text with this:
```
this is used to control highlighting of asset node in the model right after init process or by changing `assetId` prop after viewer has been initialized
```